modules/features/discover/src/main/res/layout/podcast_grid_fragment.xml

```diff
@@ -17,6 +17,8 @@
         android:id="@+id/mainNestedScrollView"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:layout_gravity=""
```

Comment:
Should this have a value?

Reply:
No, I think it got added by mistake. Removed in [113029c](https://github.com/Automattic/pocket-casts-android/pull/2214/commits/113029c2a4faa1055ca3703798bb4e2ae35d1d91).